src/api/spec/policies/comment_policy_spec.rb

```diff
@@ -67,4 +70,39 @@
       it { expect(subject).not_to permit(other_user, comment_on_request) }
     end
   end
+
+  # rubocop:disable RSpec/RepeatedExample
+  # This cop is currently not recognizing the permissions block as separate test
+  permissions :update? do
+    it 'Not logged users cannot edit comments' do
+      expect(subject).not_to permit(nil, comment)
+    end
+
+    it 'Admin cant edit other comments' do
+      expect(subject).not_to permit(admin_user, comment)
+    end
+
+    it 'Users can edit their own comments' do
+      expect(subject).to permit(comment_author, comment)
+    end
+
+    it 'Users cant edit other comments' do
+      expect(subject).not_to permit(other_user, comment)
+    end
+
+    context 'with a Nobody user comment' do
```

Comment:
Sorry for this... the `Nobody` user is just a poor naming for an anonymous user. It's something we want to refactor away, but for now, it's what we have.

So instead of referring to the `Nobody` user, we should refer to an anonymous user.

Reply:
Yes, Henne mentioned something about it the need to change this but I didn't know you have another way to refer to the user. Good to know, thank you :smile: